src/components/ThemeProvider/__tests__/themeProvider-snapshot-tests.jest.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import ThemeProvider from "../ThemeProvider";
+
+const THEME_NAME = "test-theme-name";
+
+describe("ThemeProvider renders correctly", () => {
+  it("with theme name", () => {
+    const tree = renderer
+      .create(
+        <ThemeProvider
+          theme={{
+            name: THEME_NAME
+          }}
+        >
+          <div />
+        </ThemeProvider>
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
```

Comment:
I think we can ditch the snapshot testing. we use chromatic which is much stronger, and besides it will always render `null`.

Instead, you should render the component and make sure the original children are eventually rendered there

Reply:
I think it'll come to us via sentry anyway - cause it's in our component
But I'll add it, np